app/main/browser.js

```diff
@@ -1,167 +0,0 @@
-/*
```

Comment:
We're losing a lot of history by doing this. Can't this be renamed in a commit then changed in another, if git can't figure out the move?